src/application.js

```diff
@@ -634,6 +642,45 @@ class XtkSchemaNode {
         propagateImplicitValues(this);
     }
 
+   /**
+    * Get the translation id of the label of the node
+    *
+    * @returns {string} the translation id of the label
+    */
+    getLabelTranslationId() {
```

Comment:
Why do you have both methods and public members? I think you have to choose: either public members (this.labelTranslationId...) which is more consistent with the rest of the API but has the drawback of consuming more memory (multiple ids have to be stored for each node). Or make the member variable private and dynamically compute the ids from the methods. For instance: getDescriptionTranslationId() { return this._translationId + "__@desc";  } which saves memory

Reply:
I have choose to use public members for coherence with the rest of the API